### DISCUSSION TITLE
Preserving zoom history while navigating

### DISCUSSION THREAD

**Discussion by tonyxiao** [vslinko/obsidian-zoom#83] [Ideas]
Is there a way to preserve the level of zoom setting when navigating back / forth between obsidian notes? right now when navigating back the zoom setting is lost. If not, what's the complexity in implementing something like this?

**Comment by niieani** [comment]
+1 one to this. To be clear about what is being asked here, I constantly use the "Navigate back" and "Navigate forward" shortcuts. Zoom ins and zoom outs aren't being treated as navigations by Obsidian. Not sure if that's a limitation of Obsidian or of the plugin, but it would be great to have navigation history include zooms. Thanks!
